scripts/zkSync.ts

```diff
@@ -0,0 +1,157 @@
+  const rootBundles = await spokePool.rootBundles(config.rootBundleIdToExecute);
+  console.log(`SpokePool rootBundles(${config.rootBundleIdToExecute}):`, rootBundles);
+
+  // Set allowance if depositing funds
+  const allowance = await erc20.allowance(signer.address, spokePool.address);
+  if (
+    config.amountToDepositIntoSpokePool.gt(0) &&
+    allowance.lt(config.amountToDepositIntoSpokePool) &&
+    erc20.address !== wethAddress
+  ) {
+    const txn = await erc20.approve(spokePool.address, config.amountToDepositIntoSpokePool);
+    console.log(
+      `Approving SpokePool to spend ${config.amountToDepositIntoSpokePool} of ${config.l2Erc20AddressToTest}`,
+      await txn.wait()
+    );
+  }
+  const balance = await erc20.balanceOf(signer.address);
+  const spokeBalance = await erc20.balanceOf(spokePool.address);
+  console.log(`${signer.address} ERC20 balance for ${config.l2Erc20AddressToTest}: ${balance.toString()}`);
+  console.log(`${spokePool.address} ERC20 balance for ${config.l2Erc20AddressToTest}: ${spokeBalance.toString()}`);
+
+  // Write transactions:
+
+  // Wrap ETH:
+  // const wrap = await weth.deposit({
+  //   value: config.amountToDepositIntoSpokePool.toString(),
+  //   gasPrice: "100000000",
+  //   gasLimit: "100000",
+  // });
+  // console.log(`Wrapped ETH:`, await wrap.wait())
+
+  // Deposit WETH into spoke pool so we can execute a relayer refund leaf
+  // const deposit = await spokePool
+  //   .deposit(
+  //       signer.address,
+  //       erc20.address,
+  //       config.amountToDepositIntoSpokePool.toString(),
+  //       config.depositDestinationChainId,
+  //       "1000000000000000", // relayer fee pct
+  //       currentTime.toString(),
+  //       {
+  //         value: (erc20.address === weth.address ? config.amountToDepositIntoSpokePool.toString() : "0"),
+  //         // Note: When calling contract methods using zksync-web3 with a `value`, a type error occurs because
+  //         // estimateGas() does not forward msg.value to Contracts functions. Temporary solution when calling
+  //         // payable functions in contracts is adding additional params gasPrice and gasLimit so that estimateGas()
+  //         // is not called before creating transaction.
+  //         gasPrice: "100000000",
+  //         gasLimit: "100000"
+  //       }
+  //   )
+  //   const depositReceipt = await deposit.wait()
+  //   console.log(`Deposited ${erc20.address} into SpokePool:`, depositReceipt)
+
+  const execute = await spokePool.executeRelayerRefundLeaf(
+    config.rootBundleIdToExecute,
+    {
+      // Recreating leaf constructed in  './buildSampleTree.ts'
+      amountToReturn: toBN("100000000000000000"), // 0.1
+      chainId: originChainId,
+      refundAmounts: [toBN("100000000000000000")], // -0.1
+      leafId: 0,
+      l2TokenAddress: erc20.address,
+      refundAddresses: ["0x9a8f92a830a5cb89a3816e3d267cb7791c16b04d"],
+    },
+    []
+  );
+  const executeReceipt = await execute.wait();
```

Comment:
nit: should this be commented out with the other write transactions?

Reply:
good point